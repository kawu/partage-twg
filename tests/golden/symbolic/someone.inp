1	Someone		(NP (PRO <>)):1.0
2	picked		(SENTENCE (CLAUSE (CORE (NP ) (NUC (V <>)) (NP ) (NUC (PRT ))))):1.0
3	some		(NP* (OP-DEF <>)):1.0
4	books		(NP (N <>)):1.0
5	up		(PRT <>):1.0
6	which		(NP-REL (PRO-REL <>)):1.0
7	were		(CORE* (OP-TNS <>)):1.0
8	lying		(CLAUSE (NP#WRAP# (TO-REMOVE (NP ))) (CLAUSE-PERI (PrCS (NP-REL )) (CORE (NUC (V <>))))):1.0
9	who		(NP-REL (PRO-REL <>)):1.0
10	wanted		(CLAUSE (NP#WRAP# (TO-REMOVE (NP ))) (CLAUSE-PERI (PrCS (NP-REL )) (CORE (NUC (V <>))))):1.0

