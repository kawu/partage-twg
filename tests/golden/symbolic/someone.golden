(SENTENCE (CLAUSE (CORE (NP (TO-REMOVE (NP (TO-REMOVE (NP (PRO Someone)))))) (NUC (V picked)) (NP (OP-DEF some) (N books)) (NUC (PRT up))) (CLAUSE-PERI (PrCS (NP-REL (PRO-REL which))) (CORE (OP-TNS were) (NUC (V lying)))) (CLAUSE-PERI (PrCS (NP-REL (PRO-REL who))) (CORE (NUC (V wanted))))))
(SENTENCE (CLAUSE (CORE (NP (TO-REMOVE (NP (PRO Someone)))) (NUC (V picked)) (NP (TO-REMOVE (NP (OP-DEF some) (N books)))) (NUC (PRT up))) (CLAUSE-PERI (PrCS (NP-REL (PRO-REL which))) (CORE (OP-TNS were) (NUC (V lying)))) (CLAUSE-PERI (PrCS (NP-REL (PRO-REL who))) (CORE (NUC (V wanted))))))
(SENTENCE (CLAUSE (CORE (NP (TO-REMOVE (NP (PRO Someone)))) (NUC (V picked)) (NP (OP-DEF some) (TO-REMOVE (NP (N books)))) (NUC (PRT up))) (CLAUSE-PERI (PrCS (NP-REL (PRO-REL which))) (CORE (OP-TNS were) (NUC (V lying)))) (CLAUSE-PERI (PrCS (NP-REL (PRO-REL who))) (CORE (NUC (V wanted))))))
(SENTENCE (CLAUSE (CORE (NP (TO-REMOVE (NP (PRO Someone)))) (NUC (V picked)) (NP (TO-REMOVE (NP (OP-DEF some) (N books)))) (NUC (PRT up))) (CLAUSE-PERI (PrCS (NP-REL (PRO-REL which))) (CORE (OP-TNS were) (NUC (V lying)))) (CLAUSE-PERI (PrCS (NP-REL (PRO-REL who))) (CORE (NUC (V wanted))))))
(SENTENCE (CLAUSE (CORE (NP (PRO Someone)) (NUC (V picked)) (NP (TO-REMOVE (NP (TO-REMOVE (NP (OP-DEF some) (N books)))))) (NUC (PRT up))) (CLAUSE-PERI (PrCS (NP-REL (PRO-REL which))) (CORE (OP-TNS were) (NUC (V lying)))) (CLAUSE-PERI (PrCS (NP-REL (PRO-REL who))) (CORE (NUC (V wanted))))))
(SENTENCE (CLAUSE (CORE (NP (TO-REMOVE (NP (PRO Someone)))) (NUC (V picked)) (NP (OP-DEF some) (TO-REMOVE (NP (N books)))) (NUC (PRT up))) (CLAUSE-PERI (PrCS (NP-REL (PRO-REL which))) (CORE (OP-TNS were) (NUC (V lying)))) (CLAUSE-PERI (PrCS (NP-REL (PRO-REL who))) (CORE (NUC (V wanted))))))
(SENTENCE (CLAUSE (CORE (NP (PRO Someone)) (NUC (V picked)) (NP (TO-REMOVE (NP (OP-DEF some) (TO-REMOVE (NP (N books)))))) (NUC (PRT up))) (CLAUSE-PERI (PrCS (NP-REL (PRO-REL which))) (CORE (OP-TNS were) (NUC (V lying)))) (CLAUSE-PERI (PrCS (NP-REL (PRO-REL who))) (CORE (NUC (V wanted))))))
(SENTENCE (CLAUSE (CORE (NP (PRO Someone)) (NUC (V picked)) (NP (OP-DEF some) (TO-REMOVE (NP (TO-REMOVE (NP (N books)))))) (NUC (PRT up))) (CLAUSE-PERI (PrCS (NP-REL (PRO-REL which))) (CORE (OP-TNS were) (NUC (V lying)))) (CLAUSE-PERI (PrCS (NP-REL (PRO-REL who))) (CORE (NUC (V wanted))))))

